### DISCUSSION TITLE
How to connect airflow to postgresql on 5444

### DISCUSSION THREAD

**Discussion by waardd** [apache/airflow#29508] [Q&A] [answered]
Im compossing a connect string for my airflow postgresql database

postgresql+psycopg2://{{ airflow_db_user }}:{{ airflow_db_password }}@{{ airflob_db_host }}/{{ airflow_db_name }}

But how do i fit in another port that's not 5432 but 5444

Do i use it like this?
postgresql+psycopg2://airflow_db_user:airflow_secret@myairflow.db.host:5444/airflow_dev_db

**Comment by potiuk** [comment, marked as answer]
Look here - https://docs.sqlalchemy.org/en/20/dialects/postgresql.html - you will find some examples in sqlalchemy docs (airflow uses sqlalchemy)
